draft-packages/form/KaizenDraft/Form/Primitives/InputRange/InputRange.tsx

```diff
@@ -67,6 +69,7 @@ const InputRange: React.FunctionComponent<InputRangeProps> = (
           setStep(1) // Put the stepper to 1 to avoid floating value
           onChange && onChange(e)
         }}
+        ref={ref}
```

Comment:
Traditionally I'd want to see a test here, that the external `ref` prop is passed down to right internal element. How hard would it be to add one like that (for all files here)?

Reply:
Yeah I did have a scout but we don't have any examples of testing refs. I'll have a look into how to do that as I haven't before - other than testing focus states etc.